tools/github-event-processor/RULES.md

```diff
@@ -0,0 +1,638 @@
+# Azure SDK Repository Automation Rules
+
+- [Overview](#overview)
+- [Issue Rules](#issue-rules)
+- [Pull Request Rules](#pull-request-rules)
+- [Scheduled rules](#scheduled-rules)
+
+# Overview
+
+## Nomenclature
+
+- **Trigger**: An event that occurs in GitHub to which automation should respond to.
+- **Target**: An item in GitHub that can trigger an event.  Generally, this will be an issue or pull request.
+- **Criteria**: A set of evaluations performed for a trigger target that determines which actions, if any, result.
+- **Actions**: A set of "things that happen" in response to a trigger. For example, adding a label to an issue.
+
+## Data integrations
+
+- **Service partners**: Each partner team that owns packages in our repository will need to have an association tracking a service label to one or more GitHub handles.  The contacts for service teams will follow the same pattern as the "Service Attention" process does today; it must be possible to assign a set of service owners that differs from the individuals assigned as pull request reviewers.
+
+  Example:
+
+  ```json
```

Comment:
This makes it look there is some json file that someone needs to edit in order to do this. Your example should be specific to how it works today or made more generic to avoid confusion.

Reply:
Agreed. With the next update they'll be the no-language code blocks that github MD supports with examples directly from a CODEOWNERS with the names sanitized.